dashboard/test/ui/features/star_labs/angle_helper.feature

```diff
@@ -24,50 +24,50 @@ Scenario: Free Text Input Angle Helper
 
   # defaults to 90
   Then the angle text is at "90"
-  And the Angle Helper circle is at coordinates (75,137)
+  And the Angle Helper circle is at coordinates (75,127)
```

Comment:
This test is unfortunately basically entirely magic numbers. In this case, though, I don't think there's an alternative since we're basically testing the mapping from mouse position to angle. The change here is because we have slightly changed the position of the angle picker knob (we made the picker line shorter so it doesn't overlap the new background knob).